releasenotes/notes/dx-qpd-inst-bug-fe9c50dad716b848.yaml

```
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fixed a bug in :func:`~circuit_knitting.cutting.qpd.decompose_qpd_instructions` which would
+    cause index errors if :class:`~circuit_knitting.cutting.qpd.TwoQubitQPDGate` indices were
+    specified out of order.
```

```suggestion
    not specified in strictly ascending order.
```
This is more clear to me.  What do you think?